plugins/labhub.py

```diff
@@ -92,6 +92,10 @@ def team_mapping(self):
     @cmdfilter
     def members_only(self, msg, cmd, args, dry_run):
         user = msg.frm.nick
+        if not user:
+            self.send(msg.frm, 'ERROR: The above command cannot be '
+                      'operated using nick')
```

Comment:
sentence logic error here

using -> without

Also it is not good style to have the start of the strings in different columns.
Instead, if it doesnt all fit on one line, start the string on a new line, like:
```py
            self.send(msg.frm,
                      'ERROR: The above command cannot be '
￼                      'operated using nick')
￼```

Reply:
Oh, okay. I'll make the changes

Reply:
Similar line was found at https://github.com/coala/corobo/blob/master/plugins/labhub.py#L101.

Reply:
:point_up: That was me there :upside_down_face:

Reply:
@nvzard Why thumbs down? You could've improved there :wink: